### ISSUE TITLE
Implement Split Step

### ISSUE THREAD

**Issue by iocanel** [syndesisio/syndesis#1082]
A recap after my quick chat with @iocanel: 

- @lburgazzoli  has added a step Handler for split
- on the UI side there is possibly something currently commented out
- there is no User story
- there is no corresponding UX

**Comment by paoloantinori** [comment]
This is supposed to be waiting on  UI. Do you guys confirm? @iocanel @gashcrumb

**Comment by gashcrumb** [comment]
UI or UXD?  @iocanel has the step definition added back into the store in the referenced PR.

**Comment by paoloantinori** [comment]
@gashcrumb is the comment on something currently commented out on UI still valid? or is it outdated and I should just remove it?

**Comment by gashcrumb** [comment]
On the UI side of things this change would be adding the split step back to the UI -> https://github.com/iocanel/syndesis/commit/174bc02ce6466bd30a6f1734c8e24501b827c459#diff-6c97fd18aec2ccf3b1e898b723204b58R84

**Comment by iocanel** [comment]
Since @lburgazzoli has created the step handler, the only thing missing is to uncomment the UI bits. Given that there is no UXD, I am not sure if it makes sense to do so.

I'll close this one and we can reopen, if there is a UXD.

**Comment by sjcox-rh** [comment]
Just want to confirm, is UX work requested for the Split step, or is that still a future work item?

@iocanel @gashcrumb
